mcs/class/Facades/netstandard/Makefile

```diff
@@ -15,20 +15,19 @@ SIGN_FLAGS = /delaysign /nowarn:1616,1699,618
 LIB_REFS = System System.Xml System.Xml.Linq System.Core System.Numerics System.Net.Http \
 System.IO.Compression System.ComponentModel.Composition System.IO.Compression.FileSystem
 
-LIB_MCS_FLAGS = $(SIGN_FLAGS) $(EXTRA_LIB_MCS_FLAGS)
+LIB_MCS_FLAGS = $(SIGN_FLAGS)
 
-ifneq ($(PROFILE),build)
+ifeq ($(PROFILE),build)
 # for the build profile we include stubs for these types directly in netstandard.dll
-LIB_REFS += System.Transactions System.Runtime.Serialization System.Data
-
-ifeq ($(PROFILE),xammac_net_4_5)
-LIB_REFS += System.Web.Services
+LIB_MCS_FLAGS += -d:SYSTEM_WEB_IMPLEMENTATION
+else ifeq ($(PROFILE),wasm)
+LIB_MCS_FLAGS += -d:SYSTEM_WEB_IMPLEMENTATION -unsafe
```

Comment:
this needs to add `LIB_REFS += System.Transactions System.Runtime.Serialization System.Data` to fix the build error.